examples/new-meta/tests/read_state.rs

```diff
@@ -26,7 +26,12 @@ fn read_state_bytes_with_wasm_func_returns_transformed_state() {
     assert!(META_EXPORTS_V1.contains(&FUNC_NAME));
 
     let actual_state = program
-        .read_state_bytes_using_wasm(FUNC_NAME, META_WASM_V1.to_vec(), state_args_encoded!())
+        .read_state_bytes_using_wasm(
+            Vec::<u8>::default(),
```

Comment:
state bytes should accept just vec![]?

Reply:
because i changed it to take P: Encode, it cannot just take a vec![] since it cannot infer what type should be in the vec

Reply:
> because i changed it to take P: Encode, it cannot just take a vec![] since it cannot infer what type should be in the vec

I see, but this fn accepts just vec, not `P: Encode`

Reply:
oh my bad, ill change that now